internal/web/app.go

```diff
@@ -225,7 +225,25 @@ func webSocketHandler(c echo.Context) error {
 		// Create a buffer for reading
 		buf := make([]byte, 1024)
 		for {
```

Comment:
this block was reading 1024 bytes of data and if the job was completed, it just stopped on the next iteration. Now it reads until there is no new data and then checks if the job is complete. This was if a job completes before we get to read all the data, we still print them all.